sdk/core/core-websockets/src/websockets.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Types that are compatible with NodeJS Buffer.
+ */
+export type BufferLike =
```

Comment:
Will we have a browser version too since some type `Buffer` is not available in browsers. Or should we have the common one `ArrayBufferLike`

Reply:
Yes, I'll revisit this type when I add the websocket client.